tests/PHPUnit/AbstractKernelTestCase.php

```diff
@@ -5,12 +5,7 @@
 namespace Tests\Synolia\SyliusAkeneoPlugin\PHPUnit;
 
 use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 
 abstract class AbstractKernelTestCase extends KernelTestCase
```

Comment:
We should deprecate it / or remove it from all Tests taht use it :D

Reply:
I vote to delete it :)